x11/wine-crossover/Portfile

```diff
@@ -99,32 +99,18 @@ configure.install           ${worksrcpath}/tools/install-sh
 
 configure.args              --without-alsa \
                             --without-capi \
-                            --with-cms \
-                            --with-coreaudio \
-                            --with-cups \
-                            --with-curses \
-                            --with-fontconfig \
-                            --with-freetype \
```

Comment:
I probably understand your concerns, but why is this particular change needed?

MacPorts generally tries to create as reproducible builds as possible. Though that usually means using lots of `--without-something` and `--disable-somethingelse` to make sure that libraries are not opportunistically linked. I guess the flags above don't make any real difference, but ...